renderlib/ViewerWindow.cpp

```diff
@@ -149,6 +169,9 @@ ViewerWindow::update(const SceneView::Viewport& viewport, const Clock& clock, Ge
 
   if (pickedAnything) {
     int selectionCode = gesture.graphics.getCurrentSelectionCode();
+    for (ManipulationTool* tool : sceneTools) {
+      tool->setActiveCode(selectionCode);
+    }
     forEachTool([&](ManipulationTool* tool) { tool->setActiveCode(selectionCode); });
```

Comment:
Is this line redundant now?

Reply:
No, the code is now walking two separate lists.  the forEachTool still loops over the member variable that holds a list of tools.